### PR TITLE
feat(experiments): Closes #1431 Set up local metadata parser as experiment

### LINKED ISSUE #1431: Set up the local metadata parser as experiment
Step #5 for #1017

### PR DESCRIPTION
**Pull request by sarracini**
* Set up the page scraper as an experiment!!!

### CODE REVIEW

File: experiments.json [hunk 1]
```diff
@@ -58,5 +58,19 @@
...
+  },
+  "localMetadata": {
+    "name": "Local Page Scraper",
+    "description": "Locally compute metadata for pages",
+    "control": {
+      "value": false,
+      "description": "Do not compute metadata locally"
+    },
+    "variant": {
+      "id": "exp-003-local-metadata",
```
Comment (jaredkerim) [inline experiments.json]: exp-004-local-metadata

File: experiments.json [hunk 2]
```diff
@@ -58,5 +58,19 @@
...
+      "value": true,
+      "threshold": 0.5,
```
Comment (jaredkerim) [inline experiments.json]: threshold: 0.2

File: experiments.json (hunk 1 above)
Comment (sarracini) [inline experiments.json]: whoooops

File: experiments.json [hunk 3]
```diff
@@ -58,5 +58,19 @@
       "threshold": 0.4,
       "description": "Use the score based highlights."
     }
+  },
+  "localMetadata": {
+    "name": "Local Page Scraper",
+    "description": "Locally compute metadata for pages",
```
Comment (jaredkerim) [inline experiments.json]: Please add the `active` key

**Pull request merged**